Components/templateWeather.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import {Text, View, StyleSheet} from 'react-native';
+import {Forecast} from './weather.type';
+
+interface Props {
+  forecastElement: Forecast;
+}
+
+export class TemplateWeather extends React.Component<Props> {
+  render() {
+    const forecast: Forecast = this.props.forecastElement;
+    const KtoC: number = 273;
+    const msTokmh: number = 3.6;
+
+    return (
+      <View>
+        <View>
+          <Text style={styles.textTitle}>{forecast.dt_txt}</Text>
+        </View>
+        <View style={styles.caracterictics}>
+          <View style={styles.titleCara}>
+            <View>
+              <Text style={styles.text}>
+                T°Max :{' '}
+                {Math.trunc((forecast.main.temp_max - KtoC) * 100) / 100}
+              </Text>
+            </View>
+            <View>
+              <Text style={styles.text}>
+                T°Min :{' '}
+                {Math.trunc((forecast.main.temp_min - KtoC) * 100) / 100}
+              </Text>
+            </View>
+          </View>
+          <View style={styles.titleCara}>
+            <View>
+              <Text style={styles.text}>
+                Speed Wind :{' '}
+                {Math.trunc(forecast.wind.speed * msTokmh * 100) / 100} km/h
+              </Text>
+            </View>
+            <View>
+              <Text style={styles.text}>Rain : {forecast.pop * 100}%</Text>
+            </View>
+          </View>
+          <View style={styles.titleCara}>
+            <View>
+              <Text style={styles.text}>
+                {forecast.weather && forecast.weather[0].main}
```

Comment:
Be careful here ! 

If you're sure weather exists, you can remove the condition.
Otherwise, you can use conditioinal chaining here
`forecast.weather?.[0].main`

BUT more important, happens here if your weather array is empty ?!
💣 `TypeError: Cannot read property 'main' of undefined`